numpy/typing/_array_like.py

```diff
@@ -0,0 +1,27 @@
+import sys
+from typing import Any, overload, Sequence, Tuple, Union
+
+from numpy import ndarray
```

Comment:
This is an example of what I was trying to describe in the PR description where if we do `from numpy import ndarray`, then mypy goes and looks in `__init__.pyi`, finds `ndarray`, and types it correctly. But if we were to do `from ..core.multiarray import ndarray`, then it would find no stubs for that file and fall back on treating `ndarray` as `Any`, which would be bad.